packages/babel-traverse/scripts/generators/validators.js

```diff
@@ -18,9 +18,12 @@ export interface NodePathValidators {
   }
 
   for (const type of Object.keys(virtualTypes)) {
+    const { types = [] } = virtualTypes[type];
```

Comment:
Why do we need the default value?

Reply:
Because some of the virtual-types do not have types `field`